test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -136,7 +136,7 @@ test
     await t.expect(saveBtnText).contains('Verify');
     await t.expect(pageTitle).contains('Verify with your email');
     await t.expect(challengeEmailPageObject.getFormSubtitle())
-      .contains('Check i**a@h***o.net for a verification message. Click the verification button in your email or enter the code below to continue.');
+      .contains('Check your email for a verification message. Click the verification button in your email or enter the code below to continue.');
```

Comment:
we can remove these two tests that might not make sense anymore since the message is same all the time.. ?
`challenge email authenticator screen has right labels when profile is null`
`challenge email authenticator screen has right labels when profile is empty`

Reply:
I left them in on purpose because I figured it couldn't hurt to cover a few more cases. If you think they're unnecessary, I can remove them

Reply:
Let's remove one of them, if the test(s) are redundant.

Reply:
The three test scenarios now are:
1. Profile is provided
2. Profile is null
3. Profile is empty
I'd say 2 and 3 are different enough to keep both